Features/tijdreizen.feature

```diff
@@ -0,0 +1,112 @@
+# language: nl
+
+# Nederlandse API Strategie:
+# Hoofdstuk 11 is weliswaar niet normatief, maar dient wel als uitgangspunt voor deze feature beschrijving.
+
+Functionaliteit: tijdreizen
+
+  Tijdreizen is het opvragen van resource informatie op een bepaald moment in de tijd.
+  Bij tijdreizen wordt er gekeken naar twee aspecten:
+  - welke informatie was op een bepaald moment geldig (de materiële historie)
+  - welke informatie was op een bepaald moment beschikbaar (de formele historie)
+  
+  Tijdreizen wordt mogelijk gemaakt voor de volgende resources:
+  - woonplaats
+  - openbare ruimte
+  - nummeraanduiding
+  - ligplaats
+  - standplaats
+  - verblijfsobject
+  - pand
+  - bronhouder
+  
+  Tijdreizen wordt niet ondersteund voor de volgende resources:
+  - adres
+  - adresuitgebreid
+  Dit zijn samengestelde resources, waarover geen historie wordt bijgehouden.
+
+  Tijdreizen wordt niet ondersteund bij endpoints:
+  - waar de levenscyclus van een object opgevraagd kan worden
+  - waar een specifiek voorkomen van een object opgevraagd kan worden
+
+  De volgende query parameters worden ondersteund:
+  - geldigOp: datum waarop de resource geldig is
+  - beschikbaarOp: datum en tijd waarop de resource beschikbaar is
+  
+  De waarden van geldigOp en beschikbaarOp bevatten een datum resp. datum en tijd conform ISO-8601.
+  - geldigOp bevat een datum in de vorm: YYYY-MM-DD
+  - beschikbaarOp bevat een datum en tijdstip in de vorm: YYYY-MM-DDThh:mm:ss.sss
+  
+  YYYY  Jaartal 4 cijfers
+  MM	  Maand 2 cijfers (01 = januari, etc.)
+  DD	  Dag van de maand 2 cijfers (01 to 31)
+  hh	  Uren 2 cijfers (00 to 23)
+  mm	  Minuten 2 cijfers (00 to 59)
+  ss	  Seconden 2 cijfers (00 to 59)
```

Comment:
Willen we het Engelse `to` of het Nederlandse `tot`? En zou het dan technisch gezien `tot en met` moeten zijn?